k8s/mongodb/container/mongod_entrypoint.bash

```diff
@@ -61,7 +65,8 @@ if [[ -z "${REPLICA_SET_NAME:?REPLICA_SET_NAME not specified. Exiting!}" || \
     -z "${MONGODB_IP:?MONGODB_IP not specified. Exiting!}" || \
     -z "${MONGODB_KEY_FILE_PATH:?MONGODB_KEY_FILE_PATH not specified. Exiting!}" || \
     -z "${MONGODB_CA_FILE_PATH:?MONGODB_CA_FILE_PATH not specified. Exiting!}" || \
-    -z "${MONGODB_CRL_FILE_PATH:?MONGODB_CRL_FILE_PATH not specified. Exiting!}" ]] ; then
+    -z "${MONGODB_CRL_FILE_PATH:?MONGODB_CRL_FILE_PATH not specified. Exiting!}" ||
+    -z "${STORAGE_ENGINE_CACHE_SIZE:=''}" ]] ; then
```

Comment:
The stuff inside the `${}` is different in this line, compared to the previous lines. What does it mean?

Reply:
So in the previous lines, everything is mandatory e.g. `MONGODB_PORT`, `MONGODB_FQDN` etc and we exit if anything is missing but this is an optional argument. If this is not specified we don't need to kill the container.